### PR TITLE
Test/baomt/t06 implement job detail

### PR DESCRIPTION
**Pull request by tminhbao**

### CODE REVIEW

File: src/components/CompanyCard/CompanyCard.tsx [hunk 1]
```diff
@@ -14,10 +14,15 @@ const CompanyCard = (props: Props) => {
         className="card-img-top p-3 rounded"
         alt="company__img"
         loading="lazy"
+        data-testid="company-img"
```
Comment (TruongNP) [inline src/components/CompanyCard/CompanyCard.tsx:17]: Please not use data-testis attribute. Check other position

Comment (tminhbao) [reply src/components/CompanyCard/CompanyCard.tsx]: Fixed

File: src/components/Jobs/JobDetailComponent/JobDetailComponent.tsx [hunk 2]
```diff
@@ -12,29 +12,31 @@ const JobDetailComponent = (props: Props) => {
   return (
     <div>
       <div className="row px-2">
-        <h2>{jobItem?.jobTitle}</h2>
+        <h2 data-testid="job-title">{jobItem?.jobTitle}</h2>
```
Comment (TruongNP) [inline src/components/Jobs/JobDetailComponent/JobDetailComponent.tsx:15]: Here. Please check other position

Comment (tminhbao) [reply src/components/Jobs/JobDetailComponent/JobDetailComponent.tsx]: Fixed

**Review by TruongNP — APPROVED** [review]

**Pull request merged**